test/fixtures/bn.json

```diff
@@ -0,0 +1,105 @@
+{
+  "valid": [
+    {
+      "dec": "0",
+      "hex": ""
```

Comment:
Granted, but for UI output you would surely have a padding of `1` then.  I don't see why it should be a default otherwise?